wait_spec.rb

```diff
@@ -124,15 +124,15 @@
       it "responds to Element methods" do
         decorator = browser.div.when_present
 
-        decorator.should respond_to(:exist?)
-        decorator.should respond_to(:present?)
-        decorator.should respond_to(:click)
+        expect(decorator).to respond_to(:exist?)
+        expect(decorator).to respond_to(:present?)
+        expect(decorator).to respond_to(:click)
       end
 
       it "delegates present? to element" do
         Object.class_eval do
           def present?
-            false
+            true
```

Comment:
That's the whole point of delegator, there was a pretty long discussion in #216.

For example, let's say our call to `Element#present?` is expected to return false (i.e. element is not present), but there is a chance that someone has monkey patched main object and defined `Object#present?` to always return true. Since  decorator class extends `Object`, calling `#present?` will hit `Object` rather than `method_missing`.

```ruby
class Object
  def present?
    true
  end
end

class Element
  def present?
    false
  end
end

class Decorator
  def method_missing(method)
    Element.new.send(method)
  end
end

p Element.new.present?
#=> false
p Decorator.new.present?
#=> true
```

To avoid this, we ensure that `#present?` calls are always send to `@element`, not decorator ancestors.